### PR TITLE
Indicate Lost volumes with helpers to delete the corresponding PVCs and pods

### LINKED ISSUE #435: Indicate 'Lost' volumes due to in-use drive deletions
Currently, when a ready or available drive is removed, we would `Delete` the drive object. And if the particular removed drive is replaced with a fresh drive, a new drive object will be created for the newly added one which can be formatted and used for volumes. 

When a `in-use` drive gets removed, we would tag that particular drive as `Terminating`. And the possible clean up from the application/user side would be to remove the corresponding PVCs (after cordon-ing the pods using those volumes) and when it gets recreated (by uncordoning) it would use the newly added drives for volumes.

We need a way to indicate the user about the lost volumes and suggest them to clear the corresponding PVCs.

Suggestion:
- When a in-use drive gets removed, mark the drive object as `Terminating` (already happening)
- Also, mark the corresponding volumes as `Lost` until the corresponding PVCs are removed by the user. When can also display a message to remove the corresponding PVC.

### PR DESCRIPTION
**Pull request by Praveenrajmani**
Fixes #435

### CODE REVIEW

File: pkg/utils/drive.go [hunk 1]
```diff
@@ -183,6 +189,30 @@ func DeleteDrive(ctx context.Context, driveInterface clientset.DirectCSIDriveInt
...
+				ctx, volumeName, metav1.GetOptions{TypeMeta: DirectCSIVolumeTypeMeta()},
+			)
+			if err != nil {
+				return err
+			}
+			UpdateCondition(volume.Status.Conditions,
+				string(directcsi.DirectCSIVolumeConditionReady),
+				metav1.ConditionFalse,
+				string(directcsi.DirectCSIVolumeReasonNotReady),
+				fmt.Sprintf("[LOST] Please clean up the pod [%s] and its respective PVCs in namespace [%s]", volume.Labels[directcsi.Group+"/pod.name"], volume.Labels[directcsi.Group+"/pod.namespace"]),
```
Comment (Praveenrajmani) [inline pkg/utils/drive.go:207]:
```suggestion
				fmt.Sprintf("[LOST] Please clean up the pod [%s] and its respective PVCs in namespace [%s]", volume.Labels[directcsi.Group+"/pod.name"], volume.Labels[directcsi.Group+"/pod.namespace"]),
```
```suggestion
				fmt.Sprintf("[DRIVE LOST] Please refer https://github.com/minio/direct-csi/blob/master/docs/scheduling.md"),
```

File: docs/troubleshooting.md [hunk 2]
```diff
@@ -0,0 +1,34 @@
+
+Troubleshooting
+-------------
+
+### Cleaning up abandoned volumes from a "Terminating" drive
+
+A **Terminating** drive indicates that an in-use drive has been removed and the volumes are not reachable.
+
+NOTE: If the removed drive has been replaced with a new drive, it will be in "Available" state which has to be formatted with `kubectl direct-csi drives format` to make it "Ready" for provisioning volumes in it.
```
Comment (balamurugana) [inline docs/troubleshooting.md:9]: Remove this `Note`

File: docs/troubleshooting.md [hunk 3]
```diff
@@ -0,0 +1,34 @@
...
+
+```sh
+-> kubectl direct-csi drives list --status="terminating"
```
Comment (balamurugana) [inline docs/troubleshooting.md:12]: Use `$` to indicate a command line

File: docs/troubleshooting.md [hunk 4]
```diff
@@ -0,0 +1,34 @@
...
+DRIVE      CAPACITY  ALLOCATED  FILESYSTEM  VOLUMES  NODE         ACCESS-TIER  STATUS        
+ /dev/xvdb  8.0 GiB   1.0 GiB    xfs         2        directcsi-2  -            Terminating
+```
+
+In such cases, the corresponding volumes will be indicated as follows,
+
+```sh
+-> kubectl direct-csi volumes list --drives /dev/xvdb --nodes directcsi-2 --all
```
Comment (balamurugana) [inline docs/troubleshooting.md:20]: Same as above

File: docs/troubleshooting.md [hunk 5]
```diff
@@ -0,0 +1,34 @@
...
+VOLUME                                    CAPACITY  NODE         DRIVE  PODNAME  PODNAMESPACE                                                                                           
+ pvc-ea019d52-673b-4715-a8a4-a913dd49166d  512 MiB   directcsi-2  xvdb   minio-2  default       *[DRIVE LOST] Please refer https://github.com/minio/direct-csi/blob/master/docs/scheduling.md 
+ pvc-c973cc37-01bd-4e67-abea-b1905f19fc17  512 MiB   directcsi-2  xvdb   minio-2  default       *[DRIVE LOST] Please refer https://github.com/minio/direct-csi/blob/master/docs/scheduling.md
+```
+
+To clean up the abandoned volumes and reschedule them, the respective PVCs has to be **deleted**.
+
+```sh
+-> kubectl delete pvc minio-data-3-minio-2 minio-data-1-minio-2
```
Comment (balamurugana) [inline docs/troubleshooting.md:29]: Same as above

**Review by balamurugana — APPROVED** [review]

**Review by wlan0 — APPROVED** [review]

**Pull request merged**